### ISSUE TITLE
Link tot

### ISSUE THREAD

**Issue by BastiLoe** [laroche/trading-gmbh#4]
http://pmpod.de/TradingBuchhaltungWIP.pdf -> Link ist tot

Wäre schon wenn wir das mit Erlaubnis des Autors bereitstellen könnten. Ich arbeite gerade dann die Beispielbuchungen dieser Seite als datev-Buchungsstapel zu formulieren (https://developer.datev.de/portal/de/dtvf/). Werde ich dann hier bereitstellen.

**Comment by laroche** [comment]
Ja, wäre toll, wenn das Dokument weiter verfügbar wäre. Bin da über jede
weitere Info dankbar.

Ui, es gibt eine offizielle Beschreibung zu Datev. Damit ist ja dann ein offizieller Buchungsstapel möglich.

Danke für die Infos,

Florian La Roche

**Comment by BastiLoe** [comment]
Hab den Autor im Wallstreet online Forum gefragt. Da kommt das Dokument das ich selbst nicht habe meines Wissens nach her:
https://www.wallstreet-online.de/diskussion/1318938-1-10/trading-gmbh-doppelte-buchfuehrung-termingeschaeften-co

Doku von DATEV:
https://developer.datev.de/portal/de/dtvf

Ruby Bib dazu, ich selbst bin in Python unterwegs:
https://github.com/ledermann/datev
